app/Locale/French.php

```diff
@@ -45,7 +45,7 @@ static public function get_locale()
 			'RENAME'			=> 'Renommer',
 			'RENAME_ALL'		=> 'Renommer la sélection',
 			'MERGE'				=> 'Fusionner',
-			'MERGE_ALL'			=> 'Tout fusionner',
+			'MERGE_ALL'			=> 'Fusioner la sélection',
```

Comment:
Missing a `n` at `Fusioner` => `Fusionner` :smile: 
You have the correct form LIKE THE LINE JUST ABOVE.

Reply:
Hmpf ......